tests/Microsoft.Bot.Connector.Tests/MicrosoftGovernmentAppCredentialsTests.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using System.Net.Http;
+using System.Threading.Tasks;
+using Microsoft.Bot.Connector.Authentication;
+using Microsoft.Bot.Schema;
+using Xunit;
+
+namespace Microsoft.Bot.Connector.Tests
+{
+    public class MicrosoftGovernmentAppCredentialsTests
+    {
+        [Fact]
+        public void MicrosoftGovernmentAppCredentials_Has_Gov_Endpoint()
+        {
+            var cred = new MicrosoftGovernmentAppCredentials(string.Empty, string.Empty);
+
+            Assert.Contains("login.microsoftonline.us", cred.OAuthEndpoint);
+        }
+        
+        [Fact]
+        public void MicrosoftGovernmentAppCredentials_Uses_Gov_Scope()
```

Comment:
The mostly likely "silly" (or malicious) mistake is that someone is going to change one of the URI's used (framework.gov, etc). 

Consider adding a simple check here that tests each of the key constants. If these constants are changed and we don't notice (aka: no tests fail) then we'll have a problem.

Reply:
Ok, sounds good :)